src/main/kotlin/model/BookVideoLink.kt

```diff
@@ -0,0 +1,99 @@
+package uk.gov.justice.hmpps.architecture
+
+import com.structurizr.model.Container
+import com.structurizr.model.Model
+import com.structurizr.model.Person
+import com.structurizr.model.SoftwareSystem
+import com.structurizr.view.AutomaticLayout
+import com.structurizr.view.ViewSet
+
+class BookVideoLink private constructor() {
+
+  companion object : HMPPSSoftwareSystem {
+    lateinit var model: Model
+    lateinit var system: SoftwareSystem
+    lateinit var bookVideoLinkService: Container
+
+    lateinit var vccStaff: Person
+    lateinit var omuStaff: Person
+    lateinit var courtStaff: Person
+
+    override fun defineModelEntities(model: Model) {
+      this.model = model
+
+      system = model.addSoftwareSystem(
+        "Book Video Link",
+        """
+        The Book video link service allows court users to book video conferencing rooms inside a prison to allow Prisoners to attend court sessions.
+        """.trimIndent()
```

Comment:
_nit_
Just to make this visible, I had issues with newlines in #113, so I've opened #118 to make sure we're not introducing any regressions on the weird requirements of the PlantUML converter